### ISSUE TITLE
ignoring corrupt image/label: could not convert string to float

### ISSUE THREAD

**Issue by keeper-jie** [ultralytics/ultralytics#7907]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I split coco format data into yolo format and raise error like
```
[34m[1mtrain: [0mWARNING ⚠️ /dataset/fall_detect_V1-1000_yolo/images/train/030_fea8c1dbbec720ca_24127163817.jpeg: ignoring corrupt image/label: could not convert string to float: '0,0.297796875,0.18625,0.068125,0.1775'
```
## data.yaml:
```
# Train/val/test sets as 1) dir: path/to/imgs, 2) file: path/to/imgs.txt, or 3) list: [path/to/imgs1, path/to/imgs2, ..]
path: /dataset/fall_detect_V1-1000_yolo # dataset root dir
train: images/train  # train images (relative to 'path') 128 images
val: images/val  # val images (relative to 'path') 128 images
test:  # test images (optional)

# Classes {'chair': 0 , 'sofa': 1, 'bed': 2, 'down': 3}
names:
  0: chair
  1: couch
  2: bed
  3: down
```
## train.txt/val.txt:
```
3,0.2279375,0.8831666666666667,0.087515625,0.10650000000000001
```
## code for train
```
from ultralytics import YOLO
pt_file='yolov8n.pt'
model = YOLO(pt_file)
results = model.train(data='/dataset/fall_detect_V1-1000_yolo/fall_detect.yaml', epochs=100, imgsz=640, device=[0], cache=True, batch=-1, name='coarse_detect_coco')  # results = model.train(resume=True)
```
# It seems label error, I have search google for train yolov8 in custom dataset, but no working; 

# The official doc is simple and I do not know how to load pretrained model and always download pretrain model from official

**Comment by glenn-jocher** [comment]
@keeper-jie it looks like there's an issue with the label format in your dataset. YOLOv8 expects labels to be in the format `[class, x_center, y_center, width, height]`, with each value being a float between 0 and 1, relative to the image size. The error message indicates that there's a problem converting a string to a float, likely due to the use of a comma `,` instead of a period `.` as the decimal separator.

Please ensure that your label files use periods to denote decimals and that there are no non-numeric characters or spaces. The values should be separated by spaces, not commas. For example, a correct label line should look like `3 0.2279375 0.8831666666666667 0.087515625 0.10650000000000001`.

Regarding the pretrained model, when you initialize the `YOLO` object with `pt_file='yolov8n.pt'`, it should automatically download the pretrained weights if they are not already present in your environment. Make sure you have an internet connection and sufficient permissions to download and save files in the working directory.

For more detailed guidance on training with custom datasets and other tasks, please refer to our documentation at https://docs.ultralytics.com.

**Comment by keeper-jie** [comment]
Thanks for you reply, I found the solution in https://docs.ultralytics.com/yolov5/tutorials/train_custom_data/

For a suggest, we can make a html page to show how to train custom dataset without find it in yolov5 doc.

Besides, I am Chinese and it is terrible Chinese translation in https://docs.ultralytics.com/yolov5/tutorials/train_custom_data/ 

such like train means "训练" not "火车"， we can modify it by Chinese volunteer like me and do not use machine translate.
 
Hope YOLO community better

> @keeper-jie it looks like there's an issue with the label format in your dataset. YOLOv8 expects labels to be in the format `[class, x_center, y_center, width, height]`, with each value being a float between 0 and 1, relative to the image size. The error message indicates that there's a problem converting a string to a float, likely due to the use of a comma `,` instead of a period `.` as the decimal separator.
> 
> Please ensure that your label files use periods to denote decimals and that there are no non-numeric characters or spaces. The values should be separated by spaces, not commas. For example, a correct label line should look like `3 0.2279375 0.8831666666666667 0.087515625 0.10650000000000001`.
> 
> Regarding the pretrained model, when you initialize the `YOLO` object with `pt_file='yolov8n.pt'`, it should automatically download the pretrained weights if they are not already present in your environment. Make sure you have an internet connection and sufficient permissions to download and save files in the working directory.
> 
> For more detailed guidance on training with custom datasets and other tasks, please refer to our documentation at https://docs.ultralytics.com.

**Comment by glenn-jocher** [comment]
@keeper-jie, I'm glad to hear you found the solution! We appreciate your suggestion and acknowledge the importance of clear, accessible documentation for all users. Creating a dedicated page for training custom datasets with YOLOv8 is a great idea, and we'll consider it for future updates to our documentation.

Regarding the translation issues, we truly value the feedback and the offer to help improve it. The YOLO community is global, and contributions from volunteers like you are what make it stronger and more inclusive. We'll look into improving the translations and would be grateful for any assistance you can provide. Thank you for your support, and we're committed to making the YOLO community better for everyone! 🌏🤝
